data/json/mapgen/exodii/exodii_nested_singletile.json

```diff
@@ -0,0 +1,454 @@
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_generic_tank",
+    "//": "A tank with some spare biodiesel in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": {
+          "terrain": "t_null",
+          "furniture": [ "f_standing_tank" ],
+          "liquids": { "liquid": "biodiesel", "amount": [ 200, 500 ] }
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
+    "//": "A tank with a ton of ether in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": { "terrain": "t_null", "furniture": [ "f_standing_tank" ], "liquids": { "liquid": "ether", "amount": [ 100, 200 ] } }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
+    "//": "A tank with a ton of chloroform in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": {
+          "terrain": "t_null",
+          "furniture": [ "f_standing_tank" ],
+          "liquids": { "liquid": "chem_chloroform", "amount": [ 100, 200 ] }
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
+    "//": "A tank with a ton of phenol in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": {
+          "terrain": "t_null",
+          "furniture": [ "f_standing_tank" ],
+          "liquids": { "liquid": "chem_phenol", "amount": [ 100, 200 ] }
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
+    "//": "A tank with a ton of formaldehyde in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": {
+          "terrain": "t_null",
+          "furniture": [ "f_standing_tank" ],
+          "liquids": { "liquid": "chem_formaldehyde", "amount": [ 300, 500 ] }
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
+    "//": "A tank with a ton of acid in it.",
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "X" ],
+      "terrain": { "X": "t_null" },
+      "mapping": {
+        "X": {
+          "terrain": "t_null",
+          "furniture": [ "f_standing_tank" ],
+          "liquids": { "liquid": "chem_nitric_acid", "amount": [ 100, 500 ] }
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "exodii_special_tank",
```

Comment:
Just want to be sure - when there are several `nested_mapgen_id`s with the same name, one of them is chosen randomly?

Reply:
Yep